include/dem/find_boundary_cells_information.h

```diff
@@ -225,12 +225,13 @@ class BoundaryCellsInformation
    *
    * @param boundary_cells_information A container that contains the information
    * of all the boundary cells with boundary faces
-   * @param global_boundary_cells_informationA vector that contains the geometrical
+   * @param global_boundary_cells_information A vector that contains the geometrical
```

Comment:
You re missing a parameter here which is the outlet boundaries no? Do outlet boundaries include periodic ones?

Reply:
Well see! outlet_boundaries doesn't include periodic ones. It work because I use cell->has_periodic_neighbor(face_id)` where the cell is located at outlet.